addons/udes_stock/views/stock_picking.xml

```diff
@@ -104,6 +104,20 @@
         </field>
     </record>
 
+    <!-- Stock Picking Type View -->
+    <record id="view_picking_type_form_udes_stock" model="ir.ui.view">
+      <field name="name">stock.picking.type.form.udes_stock</field>
+      <field name="model">stock.picking.type</field>
+      <field name="inherit_id" ref="stock.view_picking_type_form"/>
+      <field name="arch" type="xml">
+        <xpath expr="//form/sheet/group[last()]" position="after">
+          <group string="Empty Pickings" groups="base.group_no_one">
```

Comment:
Can anyone think of a better name for this feature?

Reply:
Maybe just `Pickings`? We're putting this in the `udes_stock` module so everything here is pretty core and generic. It's likely we'll have other config later on to put in that doesn't nicely fall into other areas.